src/pages/SplitPosition/index.tsx

```diff
@@ -47,15 +49,23 @@ export const SplitPosition = () => {
         connect()
       }
     },
-    [status, CTService, connect, collateralToken]
+    [status, CTService, connect]
   )
 
+  // TODO Fix this
+  if (!collateralToken) {
+    return null
+  }
```

Comment:
Is this a missing TODO?

Reply:
No, it is not. But I wanted to check for null (and reuse `useCollateral` that returns a Maybe) to avoid doing that in subcomponents. Maybe another option is to throw an error or log something.